std_msgs/Header header
uint32 sending_agent
DetectedObject[] objects